### PR TITLE
#8865: Update reference dispatch times. Move tt_lib to ttnn.

### PR DESCRIPTION
**Pull request by nemanjagrujic**

### CODE REVIEW

**Review by boris-drazic — APPROVED** [review]

File: tests/ttnn/profiling/ops_for_profiling.py [hunk 1]
```diff
@@ -584,7 +584,7 @@ def unary_div_bw(x, y):
         "op": ttnn.add,
         "name": "ttnn.complex_add",
         "is_complex": [True, True],
-        "need_out_mem_cfg": True,
+        # "need_out_mem_cfg": True,
```
Comment (ayerofieiev-tt) [inline tests/ttnn/profiling/ops_for_profiling.py:587]: remove?

Comment (nemanjagrujic) [reply tests/ttnn/profiling/ops_for_profiling.py]: Removed

Comment (nemanjagrujic) [reply tests/ttnn/profiling/ops_for_profiling.py]: @ayerofieiev-tt Can you checkout if pull req is ok now?

**Review by eyonland — APPROVED** [review]
I'm approving but I do see we have a problem with guarding our performance per op.   This is something we need to solve.  I will be bringing this up to my core team.

**Comment by nemanjagrujic** [comment]
> I'm approving but I do see we have a problem with guarding our performance per op. This is something we need to solve. I will be bringing this up to my core team.

@eyonland Yes, it would be great if we had these measurements in pipeline.

**Pull request merged**